static/elements/chromedash-feature.html

```diff
@@ -84,6 +84,11 @@
             <iron-icon icon="icons:extension" class="experimental"></iron-icon>
           </span>
         </template>
+        <template is="dom-if" if="{{experimentalframework}}">
```

Comment:
instead of a defining a new prop, you can just use `feature.meta. experimentalframework` directly.